src/foraging_gui/Foraging.py

```diff
@@ -3766,6 +3766,22 @@ def _set_default_project(self):
             self.Metadata_dialog.project_info = project_info
             self.Metadata_dialog.ProjectName.addItems([project_name])
         return project_name
+    
+    def _empty_initialize_fields(self):
+        '''empty fields from the previous session'''
+        self.open_ephys=[]
+        self.ManualWaterVolume=[0,0]
+        # clear camera start and end time
+        if not self.Camera_dialog.StartRecording.isChecked():
+            self.Camera_dialog.camera_start_time=''
+            self.Camera_dialog.camera_stop_time=''
+        # clear fiber start and end time
```

Comment:
I don't quite know the downstream effect but would these variables be better initialized as None? seems weird to me that they are just empty stings but maybe that is what the code is expecting downstream

Reply:
Also does this fix #1124?